docs/BEST_PRACTICES.md

```diff
@@ -30,6 +30,7 @@ Good:
 Bad:
  - 2017-01-01 00:00:00
 
+When parsing datetimes or converting between timezones, the library accepted for use by singer taps is pytz. This project has more accurate timezone data than the builtin datetime.timezone.
```

Comment:
Wrapping and `BACKTICKpytzBACKTICK`.

Reply:
Done. Note there is inconsistent styling on wrapping in this greater document.